build.sh

```diff
@@ -406,6 +413,24 @@ function build_android {
 
     build_desktop "${MOBILE_HOST_TOOLS}"
 
+    # When building the samples, we need to partially "install" the host tools so Gradle can see
```

Comment:
Why not do an install then?

Reply:
Doing a full `install` builds everything, which is wasteful since we only need the host tools. What we want is something like `ninja install matc cmgen`, but that doesn't exist.

Reply:
Makes sense 👍